### PR TITLE
alpine

### PR DESCRIPTION
**Pull request by frkr**

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@frkr Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud-samples/eureka&pullRequestId=29)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-cloud-samples/eureka&pullRequestId=29) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@frkr Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud-samples/eureka&pullRequestId=29)!

File: src/main/docker/Dockerfile [hunk 1]
```diff
@@ -1,6 +0,0 @@
-FROM java:7
-VOLUME /tmp
-ADD eureka-0.0.1-SNAPSHOT.jar /app.jar
-RUN bash -c 'touch /app.jar'
-EXPOSE 8761
-ENTRYPOINT ["java","-Djava.security.egd=file:/dev/./urandom","-jar","/app.jar"]
```
Comment (spencergibb) [inline src/main/docker/Dockerfile]: You've missed part of the entrypoint

Comment (frkr) [reply src/main/docker/Dockerfile]: I did not.

The touchy its useless. The volume can be made outside. And the random thing its to address a bug already solved.

Comment (spencergibb) [reply src/main/docker/Dockerfile]: Explaining those kinds of things upfront in a PR is helpful. Where is the 'random thing' solved?

Comment (frkr) [reply src/main/docker/Dockerfile]: its resolved here: https://bugs.java.com/view_bug.do?bug_id=6202721

Comment (spencergibb) [reply src/main/docker/Dockerfile]: https://wiki.apache.org/tomcat/HowTo/FasterStartUp#Entropy_Source

Comment (spencergibb) [reply src/main/docker/Dockerfile]: it even mentions the bug you linked to. It solved the funny `/./` syntax, not the shortage of entropy.

Comment (frkr) [reply src/main/docker/Dockerfile]: Look like over best pratices. Maybe let that on java opts env options?

Comment (spencergibb) [reply src/main/docker/Dockerfile]: the random entropy jar option needs to stay, otherwise, I'll go ahead and close this.

Comment (frkr) [reply src/main/docker/Dockerfile]: Ok. Close. I will make another pr. Tx

**Pull request closed without merging**